tests/trainer/test_fsdp_checkpoint.py

```diff
@@ -971,19 +970,18 @@ def test_elastic_resumption(
     autoresume: bool,
     precision: str,
     sharding_strategy,
-    s3_bucket,
-    s3_read_only_prefix,
+    uc_volume_read_only,
     num_shards: int,
 ):
     if autoresume:
         run_name = 'my-autoresume-run'
     else:
         run_name = None
 
-    base_path = (
-        f's3://{s3_bucket}/{s3_read_only_prefix}/elastic_test/'
-        f'{sharding_strategy.lower()}_sharded_{precision}_'
-        f'{num_shards}/'
+    base_path = os.path.join(
```

Comment:
Same issue with `.metadata` not existing (daily tests that fail: https://github.com/mosaicml/composer/actions/runs/15356296375/job/43215949886) cc: @ethantang-db, can you lmk when you move this files? Thanks!